README.md

```diff
@@ -115,14 +115,30 @@ Behat tests can be executed from the repository root by running:
 $ ./bin/behat -c tests/behat.yml
 ```
 
-With a single Phing task, you can run every tests suites:
```

Comment:
Why the "./bin/phing behat" command is removed?. 
Maybe we can check if this command is used in another place (started kit, workflow of release, ... ) and remove the code on the platform.